### ISSUE TITLE
Adjustment of gas properties based on ideal gas law

### ISSUE THREAD

**Issue by olavosilva** [open-pulse/OpenPulse#23]
Updating of gas acoustic properties with temperature and pressure.
@dmtuozzo @JacsonGilVargas

**Comment by raphaeltimbo** [comment]
Hi there, @olavosilva ! 
If needed, I have a code that implements the real gas Peng-Robinson model.

**Comment by olavosilva** [comment]
Great, @raphaeltimbo! Could you share it with us? @dmtuozzo @JacsonGilVargas

**Comment by raphaeltimbo** [comment]
[state.zip](https://github.com/open-pulse/OpenPulse/files/5270327/state.zip)
The code was written for python 2.7, so you guys might have to change some parts of it.
Also, there are some parts of the code that are in portuguese, maybe we could change that in the future if you decide to use the code.

The basic use is something like:
```python
# create a thermodynamic state
st = State(p=100000, T=300, comp=dict(N2=0.9, O2=0.1))

# get desired properties using methods
rho = st.rho()
enthalpy = st.h()

**Comment by olavosilva** [comment]
Thanks, @raphaeltimbo!

**Comment by raphaeltimbo** [comment]
This is a more simple (and limited) approach to have a real gas model implemented.
If in the future you need a more robust solution, we would need to use softwares such as REFPROP/CoolProp. We also have a code for calculating centrifugal compressors performance that implements this solution, but as I said, it would be a more complex solution and it would require having more dependencies in the openpulse project.

**Comment by olavosilva** [comment]
Solved in #109
